pubspec.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
     sdk: flutter
   async: ^2.0.7
 environment:
-    sdk: ">=2.1.0 <3.0.0"
+    sdk: '>=2.1.0-dev.68.0 <3.0.0'
```

Comment:
Thank you for the pull request! 
But I don't really get it. Is `2.1.0-dev.68.0` considered smaller than `2.1.0`?

Reply:
Actually you maybe right -- but since that is what the current SDK level is (well its now dev-69 ?? or so) I think we need to match that and then when 2.1.0  is released we can modify it?  I have this use of SEMVAR .. so confusing.. this has caused issues in the past .. surprised they did this again..